### ISSUE TITLE
Build fails on Ubuntu due to /usr/lib/libGL.so paths

### ISSUE THREAD

**Issue by mmorehea** [janelia-flyem/NeuroProof#6]
In Ubuntu, libGL.so, libICE.so, libX11.so, etc, are all stored in /usr/lib/x86_64-linux-gnu/, not /usr/lib64/.

I makde a quick soft link with 

```
sudo ln -s /usr/lib/x86_64-linux-gnu/ /usr/lib64
```

and it builds to completion.

**Comment by stuarteberg** [comment]
Yeah, sorry about that. This is a known issue. (Our [CI script](https://github.com/janelia-flyem/NeuroProof/blob/master/circle.yml#L21-L22) uses the same workaround that you found.)  For future users, I've now added a note about this in the README, but I'll leave this issue open until we come up with a proper fix.
